llvm/lib/Target/WebAssembly/TargetInfo/WebAssemblyTargetInfo.cpp

```diff
@@ -29,8 +29,9 @@ Target &llvm::getTheWebAssemblyTarget64() {
 
 extern "C" LLVM_ABI LLVM_EXTERNAL_VISIBILITY void
 LLVMInitializeWebAssemblyTargetInfo() {
-  RegisterTarget<Triple::wasm32> X(getTheWebAssemblyTarget32(), "wasm32",
-                                   "WebAssembly 32-bit", "WebAssembly");
+  RegisterTarget<Triple::wasm32, /*HasJIT=*/true> X(
```

Comment:
I'm not sure what `HasJIT` means exactly here, but wasm doesn't support JIT in the traditional sense.. at least you cannot write new instructions to memory at runtime.

Reply:
I do not have any issues there -- anything that makes the tests run on that platform would be a good first step.

Reply:
> do you have any issues with me skipping the hostsupportsjit check for the tests 

Yes we can start with that.